src/main/java/school/lemon/changerequest/java/io/hw1/App.java

```diff
@@ -0,0 +1,80 @@
+package school.lemon.changerequest.java.io.hw1;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.FileNotFoundException;
+
+/**
+ * Created by lera on 22.01.17.
+ */
+public class App {
+    public static void main(String[] args) throws FileNotFoundException {
+        FileManager manager = new FileManager(".");
+
+        System.out.println(manager.ls());
+        System.out.println();
+        System.out.println(manager.ls(Order.BY_LAST_UPDATE_DESCENDING));
+        System.out.println();
+        System.out.println(manager.ls(Order.BY_LEXICOGRAPHICAL_DESCENDING, false));
+        System.out.println(manager.filter(Order.BY_LEXICOGRAPHICAL_DESCENDING, true, new FileFilter() {
+            @Override
+            public boolean accept(File pathname) {
+                return pathname.getName().endsWith("gradle");
+            }
+        }));
+
+        System.out.println("Try to create file, that already exist.");
+        manager.createFile("LICENSE");
+
+        System.out.println("Create new file.");
+        manager.createFile("newFile");
+
+        System.out.println("Try to create directory, that already exist.");
+        manager.createDirectory("gradle");
```

Comment:
Please wrap method that should throw exception with try-catch. Demo app shouldn't fail with any exception.